.gitignore

```diff
@@ -1,3 +1,4 @@
+./apps/cmake-build-debug
+/build
```

Comment:
It would be nice if you follow the convention...
The directories should be prefixed with ./